unitTests/ui/PagerTest.ts

```diff
@@ -245,6 +245,18 @@ export function PagerTest() {
           done();
         });
       });
+
+      it(`when having a resultPerPage that is not a divider of maximumNumberOfResultsFromIndex 
+      should prevent requesting more result than the maximumNumberOfResultsFromIndex option`, () => {
+        const resultsPerPage = 12;
+        const pageNumber = 84; // the last page for 1000 results with a resultsPerPage of 12
```

Comment:
Could we remove the comment if we simplified the numbers and set `maximumNumberOfResultsFromIndex` to a smaller value? I think it would make the test easier to understand and validate.

Reply:
yes but the test would be longer because we would need to reinitialize the component